examples/simple/max-restarts.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: jobset.x-k8s.io/v1alpha1
+kind: JobSet
+metadata:
+  name: failurepolicy
+spec:
+  # On failure, restart all jobs up to 3 times.
+  failurePolicy:
+    maxRestarts: 3
+  replicatedJobs:
+  - name: worker1
```

Comment:
I would avoid naming different replicatedJobs worker1 and worker2, they should have distinct names to indicate that multiple replicatedJobs are needed only when they are sufficiently different in what they do.

Reply:
makes sense, done